tiktoken-rs/src/api.rs

```diff
@@ -328,3 +306,70 @@ mod tests {
         assert!(max_tokens > 0);
     }
 }
+
+#[cfg(feature = "async-openai")]
+pub mod async_openai_api {
+    use anyhow::Result;
+    use async_openai::types::ChatCompletionRequestMessage;
+
+    pub fn async_openai_num_tokens_from_messages(
```

Comment:
We already publish a module here, no need to duplicate it in the function name.
```suggestion
    pub fn num_tokens_from_messages(
```